More Text options SlideView Textview

dscottpi opened the pull request:
Hey! 

Just a couple of changes that I needed on a project that I thought might be useful for others. I've added the ability to: 

- Change the text size of the SlideView's TextView

- Center the SlideView's TextView

Thanks for this library! It saved us a lot of time.

kizitonwose commented:
Hi @dscottpi Nice PR, I like the idea of customizing the text but I really don't like the idea of putting the text on the center of the view. If you can create another PR without that change or add a commit that removes the change directly to your branch, I'd gladly merge that. Also, maybe you can add a getter for the TextView. That way, users can do other text customizations like custom fonts e.t.c

dscottpi commented:
@kizitonwose Yeah I totally understand. I removed the centre text option. Giving access to the textview makes a lot of sense, I added a method for getting it 😄 

Thanks!

kizitonwose commented:
@dscottpi Thanks for the changes!
